app/views/local_transaction/unavailable_service.html.erb

```diff
@@ -1,34 +1,42 @@
 <%= render layout: 'shared/base_page', locals: {
-  title: @service.title,
+  title: t('formats.local_transaction.service_not_available', country_name: @country_name),
   publication: @publication,
   edition: @edition,
 } do %>
   <div class="interaction">
     <p class="govuk-body">
-      We've matched the postcode to <span class="local-authority"><%= @local_authority.name %></span>.
+      <%= t('formats.local_transaction.matched_postcode_html', local_authority: @local_authority.name) %>
     </p>
 
-  <% if @service.body.present? %>
-    <p class="govuk-body"><%= @service.body %></p>
-  <% end %>
-
-    <p id="get-started" class="get-started group">
-      <%= render "govuk_publishing_components/components/button", {
-        text: @service.button_text,
-        rel: "external",
-        margin_bottom: true,
-        href: @service.button_link,
-      } %>
+    <p class="govuk-body">
+      <%= t('formats.local_transaction.service_not_available', country_name: @country_name) %>.
     </p>
-  </div>
 
-  <% if @publication.more_information.present? %>
-    <section class="more">
-      <div class="more">
-        <%= render "govuk_publishing_components/components/govspeak", {} do %>
-          <%= raw @publication.more_information %>
-        <% end %>
+    <% if @local_authority.url.present? %>
+      <p class="govuk-body">
+        <%= t('formats.local_transaction.find_other_services') %>
+      </p>
+      <div class="local-authority-result"
+           data-module="auto-track-event"
+           data-track-category="postcodeSearch:local_transaction"
+           data-track-action="postcodeResultShown:devolvedAdministrationUnavailable"
+           data-track-label=<%= @country_name %>>
```

Comment:
This event should fire regardless of whether there is a local_authority url present or not, so should be set before the conditional. (label needs quotes too)

Reply:
👍